packages/console/src/components/AppContent/index.tsx

```diff
@@ -23,6 +25,8 @@ const AppContent = () => {
   const location = useLocation();
   const navigate = useNavigate();
   const { firstItem } = useSidebarMenuItems();
+  const mainRef = React.useRef<HTMLDivElement>(null);
```

Comment:
nit: remove `React.`, and import from top

Reply:
updated.